### PR TITLE
redirect animation

### PR DESCRIPTION
**Pull request by MauriceMiot**

### CODE REVIEW

**Comment by the-code-workers** [comment]
Your Code is succesfully Success

File: src/components/RedirectPage/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+._title{
+  font-size: 1rem;
+  @include font('helvetica', 'helvetica17');
+  margin-top: 1.5rem;
+  text-align: center;
+}
+
+._spinnerContainer{
+align-self: center;
```
Comment (rotcax) [inline src/components/RedirectPage/styles.module.scss]: Please ident the property

Comment (MauriceMiot) [reply src/components/RedirectPage/styles.module.scss]: is solved

**Comment by the-code-workers** [comment]
Your Code is succesfully Success

**Review by rotcax — APPROVED** [review]

**Pull request merged**